doc/plugins/proxy-cache.md

```diff
@@ -31,7 +31,7 @@ The proxy-cache plugin, which provides the ability to cache upstream response da
 
 | Name               | Type           | Requirement | Default                   | Valid                                                                           | Description                                                                                                                                                                                                                                  |
 | ------------------ | -------------- | ----------- | ------------------------- | ------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| cache_zone         | string         | required    |                           |                                                                                 | Specify which cache area to use, each cache area can be configured with different paths. In addition, cache areas can be predefined in conf/config.yaml file                                                                                 |
+| cache_zone         | string         | optional    |  disk_cache_one           |                                                                                 | Specify which cache area to use, each cache area can be configured with different paths. In addition, cache areas can be predefined in conf/config.yaml file                                                                                 |
```

Comment:
We need to tell people the cache will be invalid if `cache_zone` do not match.

Reply:
Nice, this sounds great.